sydent/db/invite_tokens.sql

```diff
@@ -24,7 +25,8 @@ CREATE TABLE IF NOT EXISTS invite_tokens (
     received_ts bigint, -- When the invite was received by us from the homeserver
     sent_ts bigint, -- When the token was sent by us to the user
     origin_id integer, -- original id in homeserver's DB that this was replicated from (if applicable)
-    origin_server text -- homeserver this was replicated from (if applicable)
+    origin_server text, -- homeserver this was replicated from (if applicable)
+    valid_until_ms integer not null default 0 -- When the invite is due to expire
```

Comment:
could you decide if it's an expiry time (which implies it is invalid *at* the given timestamp) or a "valid until" time (which implies it is valid at that time)?